### PR TITLE
Update i18n guide to accompany docs PR 9896

### PR DESCRIPTION
**Pull request by delucis**
This PR updates the internationalization guide to accompany https://github.com/withastro/docs/pull/9896

This won’t be the last change here because nav config will also change later, but wanted to do this incrementally nonetheless. It does mean things are _slightly_ in a half-and-half state where the updated section here doesn’t really mention the `nav.ts` files anymore but I think that’s OK given we’ll be revisiting soon.

### CODE REVIEW

**Review by sarah11918 — APPROVED** [review]
Thank you @delucis ! I made some light edits in IDX because I couldn't figure out how to make them be suggestions. You are welcome to keep/ignore as you wish, and I'll try to figure out editing PRs in IDX properly!

**Comment by trueberryless** [comment]
I was very confused why the Hacktoberfest banner was still there (#29 hehe), but this explains it pretty good:

image

File: src/content/docs/guides/i18n.mdx [hunk 1]
```diff
@@ -56,17 +56,68 @@ You can also visit GitHub directly and go through your language's [open Pull Req
...
+```yml title="src/content/i18n/en.yml"
+a11y.sectionLink: Section titled
+# Site settings
+site.title: Astro Documentation
+# Left Sidebar
+leftSidebar.sponsoredBy: Sponsored by
+# ...
+```
+
+Keys are not translated, only the English text. So the example above is translated into French is:
```
Comment (delucis) [inline src/content/docs/guides/i18n.mdx]: @sarah11918 Aren’t the keys also in English? 😅

Comment (casungo) [inline src/content/docs/guides/i18n.mdx:109]: Maybe something like this to further clarify?

```suggestion
Keys, although they are written in English, should not be translated. The only part that should be translated is the text following the key. For example, when translated into French, it appears like this:
```

Comment (casungo) [reply src/content/docs/guides/i18n.mdx]: Maybe something like this to further clarify?

```suggestion
Keys, although they are written in English, should not be translated. The only part that should be translated is the text following the key. For example, when translated into French, it appears like this:
```

Comment (delucis) [reply src/content/docs/guides/i18n.mdx]: I like that! But I do think it’s a bit much when just explaining a the example. We can probably simplify back to something like the original:
```suggestion
Keys are not translated, so the example above is translated into French as:
```

**Pull request merged**